src/server/scripts/EasternKingdoms/zone_undercity.cpp

```diff
@@ -3974,10 +3983,6 @@ class npc_lady_sylvanas_windrunner_bfu : public CreatureScript
             {
                 switch (eventId)
                 {
-                    case EVENT_SUMMON_SKELETON:
-                        DoCast(me, SPELL_SUMMON_SKELETON);
```

Comment:
Can probably remove the enums for the event and spell as well, seeing as it looks like (at least at a glance) they're going unused.